Spec/AblySpec-Bridging-Header.h

```diff
@@ -2,7 +2,7 @@
 //  Use this file to import your target's public headers that you would like to expose to Swift.
 //
 
-#include <ably/ably.h>
+#include "Ably.h"
```

Comment:
But I prefer to remove the prefix dependency. It helps when you change de Pod name.